platform/features/timeline/bundle.js

```diff
@@ -127,7 +127,16 @@ define([
                             14400000,
                             28800000,
                             43200000,
-                            86400000
+                            86400000,
+                            86400000 * 2,
+                            86400000 * 5,
+                            86400000 * 10,
+                            86400000 * 20,
+                            86400000 * 30,
+                            86400000 * 60,
+                            86400000 * 120,
+                            86400000 * 240,
+                            86400000 * 365
```

Comment:
Do we need to enumerate all of the possible zoom states, or could we just have a max and a min value and double/halve the current value on each step?